recipes/lumispy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lumispy" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lumispy-{{ version }}.tar.gz
+  sha256: 1f517c85cd02ffcebea3611d8f98edc769441d1f56b8d819995a968e5a6dd502
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - hyperspy-base >=1.5.2
+    - numpy
+    - python
+    - scipy
+
+test:
+  imports:
+    - lumispy
+    - lumispy.components
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/lumispy/lumispy
+  summary: Luminescence spectroscopy data analysis with HyperSpy.
+  dev_url: https://github.com/lumispy/lumispy
+  license: GPL-3.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - jlaehne
+    - jordiferrero
```

Comment:
@jlaehne, @jordiferrero, can you please confirm that you are willing to be a recipe maintainer?

Reply:
Yes, I agree.

Reply:
I agree